_faq/14-app-context-vs-singleton.asciidoc

```diff
@@ -0,0 +1,37 @@
+= What's the difference between `@ApplicationScoped` and `@Singleton`
+
+Both these contexts behave very similarly.
+A single bean instance is created and shared across the application.
+However, there are some important differences.
+
+== Client Proxy
+
+First of all, `@ApplicationScoped` is a *normal* scope whereas `@Singleton` is a *pseudo-scope* (using the CDI terminology).
+What does it mean?
+In the first place, for normal scopes a https://docs.jboss.org/cdi/spec/2.0/cdi-spec.html#client_proxies[client proxy] is always injected.
+This is a container construct that delegates all method calls to the current bean instance.
+Looks like some unnecessary overhead but in fact allows the container to do some useful stuff.
+
+== Serialization
+
+The client proxies are serializable even when the bean itself may not be.
+And so you can `@Inject` a normal-scoped bean into a bean with a passivating scope (such as `@SessionScoped`).
+See https://docs.jboss.org/weld/reference/latest/en-US/html/scopescontexts.html#_the_singleton_pseudo_scope[Weld Reference Guide] for more information.
+
+== Lazy Creation
+
+The container may initialize the bean instances of normal scoped beans lazily. 
```

Comment:
Leave out `may`, that's just how we do it.
`The container initializes normal scoped bean instances lazily.`
Or maybe better:
`Having a client proxy allows Weld to initialize all normal scoped beans lazily.`